pom.xml

```diff
@@ -174,6 +174,8 @@
          IMPORTANT: always explain the reason for overriding the plugin version! -->
     <!-- Surefire version 2.19.1 coming from parent is buggy and kie-osgi tests in droolsjbpm-integration are failing because of that. -->
     <version.surefire.plugin>2.18.1</version.surefire.plugin>
+    <version.enforcer.plugin>3.0.0-M1</version.enforcer.plugin> <!-- Needed to support JDK 9 -->
+    <version.javadoc.plugin>3.0.0-M1</version.javadoc.plugin> <!-- Needed to support JDK 9 -->
```

Comment:
Can we add something in the comment as to when will we be able to remove these overrides?

We depend on ipbom 7.0 which extends jboss parent 22.
but ipbom 8.0 already extends jboss parent 25, which might already be JDK 9 compliant (as WildFly uses it too)

Reply:
I will add a comment, but we should find a better way to detect these. E.g. have a script that checks which property values are exactly the same as in parent, so can be safely removed. But that will of course require some time to implement.